docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -185,8 +185,8 @@ You can then use `/tmp/isrgrootx1.pem` as the value of the `db_service.databases
 
 ## Step 3/4. Configure Atlas
 
-### Enable self-managed X.509 authentication
-
+<Tabs>
```

Comment:
The introduction paragraph still says that this guide will show you how to configure self-managed X.509 authentication. We should probably also mention there that using IAM authentication is an option.

Reply:
I've updated it to not specifically mention any authentication method.